templates/base/Dockerfile

```diff
@@ -1,17 +1,19 @@
-FROM --platform=${TARGETPLATFORM} ubuntu:jammy AS ubuntu-jammy
+FROM --platform=${TARGETPLATFORM} ubuntu:oracular AS ubuntu-oracular
```

Comment:
Hey @zsoltbalogh and @Akos-Kreutz, why pick Oracular instead of Noble (LTS)?

Reply:
Hi @natocesarrego I discussed this with @zsoltbalogh and we concluded that the easiest way forward is to have the newest version always, so we always have the most secure packages which the LTS usually don't provide in the long-term.